packages/core/scss/components/action-sheet/_layout.scss

```diff
@@ -124,6 +126,10 @@
     // Actionsheet footer
     .k-actionsheet-footer {
         flex: none;
+        border-width: 0;
+        border-top-width: $kendo-actionsheet-footer-border-width;
+        border-style: solid;
+        border-color: transparent;
```

Comment:
The border-color should be defined in the _theme.scss file

Reply:
exactly, I don't think that we need transparent border here